modules/apps/27-interchain-accounts/types/account.go

```diff
@@ -37,6 +45,20 @@ func GenerateAddress(moduleAccAddr sdk.AccAddress, portID string) sdk.AccAddress
 	return sdk.AccAddress(sdkaddress.Derive(moduleAccAddr, []byte(portID)))
 }
 
+// ValidateAccountAddress performs basic validation of interchain account addresses, enforcing constraints
+// on address length and character set
+func ValidateAccountAddress(addr string) error {
```

Comment:
Moved `ValidateAccountAddress` from `version.go` to `account.go` along with the associated pkg level vars: `isValidAddr` and `DefaultMaxAddrLength`.

The `version.go` file has been deleted